apiserver/facades/controller/migrationtarget/migrationtarget_test.go

```diff
@@ -51,6 +52,9 @@ type Suite struct {
 var _ = gc.Suite(&Suite{})
 
 func (s *Suite) SetUpTest(c *gc.C) {
+	ctrl := gomock.NewController(c)
```

Comment:
Use a `setupMocks` method as mentioned prior.

Reply:
yes of course, done

Reply:
done